### PR TITLE
Allow input shape override to broadcast across all inputs

### PR DESCRIPTION
**Pull request by mgoin**
For example, many BERT models have 3 inputs with exactly the same shape.
So instead of writing 
`deepsparse.benchmark bert.onnx --input_shapes [1,128],[1,128],[1,128]`
you can now write
`deepsparse.benchmark bert.onnx --input_shapes [1,128]`
and it will try to apply the shape to all inputs.

### CODE REVIEW

File: src/deepsparse/utils/onnx.py [hunk 1]
```diff
@@ -229,14 +229,31 @@ def override_onnx_input_shapes(
...
-    assert len(external_inputs) == len(input_shapes)
-
     # Input shapes should be a list of lists, even if there is only one input
     assert all(isinstance(inp, list) for inp in input_shapes)
 
-    # Overwrite the shapes
+    # If there is a single input shape given and multiple inputs,
+    # duplicate for all inputs to apply the same shape
+    if len(input_shapes) == 1 and len(external_inputs) > 1:
+        for i in range(len(external_inputs) - 1):
```
Comment (rahul-tuli) [inline src/deepsparse/utils/onnx.py:238]: Why do we leave out the last external input here?

Comment (rahul-tuli) [inline src/deepsparse/utils/onnx.py:238]: nit
```suggestion
        for _ in range(len(external_inputs) - 1):
```

**Review by bfineran — DISMISSED** [review]
nice

**Review by rahul-tuli — DISMISSED** [review]

File: src/deepsparse/utils/onnx.py (hunk 1 above)
Comment (mgoin) [reply src/deepsparse/utils/onnx.py:238]: This is trying to duplicate input_shapes to the same size as external_inputs, so i had it do this external_inputs-1 times. I'm not a fan of this loop so cleaner suggestions would be nice

Comment (ElizaWszola) [reply src/deepsparse/utils/onnx.py]: Maybe something like
`input_shapes.extend([input_shapes[0] for i in range(1, len(external_inputs))])` ?

**Comment by ElizaWszola** [comment]
Looks good

**Review by ElizaWszola — DISMISSED** [review]
Looks good

File: src/deepsparse/utils/onnx.py (hunk 1 above)
Comment (mgoin) [reply src/deepsparse/utils/onnx.py]: Cool, that works and I like it better

**Review by andy-neuma — APPROVED** [review]
cool

**Review by bfineran — APPROVED** [review]

**Pull request merged**